app/src/main/res/layout/item_susi_piechart.xml

```diff
@@ -11,8 +11,8 @@
     app:cmv_arrowPosition="left"
     app:cmv_backgroundColor="@color/other_message_bg"
     app:cmv_backgroundColorPressed="@color/other_message_bg_dark"
-    app:cmv_contentPadding="3dp"
-    app:cmv_cornerRadius="3dp"
+    app:cmv_contentPadding="10dp"
```

Comment:
@betterclever Is it possible for us to move these dimensions to dimens.xml instead of hard coding them?